.github/workflows/ci.yml

```diff
@@ -69,4 +72,5 @@ jobs:
           connection-string-name: AzureServiceBus_ConnectionString
           tag: ASBTransport
       - name: Run tests
-        uses: Particular/run-tests-action@v1.0.0
+        run: dotnet test src --configuration Release --no-build --framework ${{ matrix.framework }} --logger "GitHubActions;report-warnings=false" -m:1
```

Comment:
I wonder if we should add the ability to specify a framework to the run tests action? Main benefit I can think of would be to make this look more consistent with other repos.